src/kpatch_user.c

```diff
@@ -368,6 +368,60 @@ storage_have_patch(kpatch_storage_t *storage, const char *buildid,
 	return PATCH_FOUND;
 }
 
+/* TODO(pboldin): cache these */
+static char *
+storage_get_description(kpatch_storage_t *storage,
```

Comment:
As we only print `description`s for latest patches available place `description` in the `struct kpatch_storage_patch`.